src/analysis/Environments/Environments.ts

```diff
@@ -248,10 +245,11 @@ type AppWithWorkspace = DecoratedResourceAttributes & App;
 type DecoratedComputeResource = RuntimeWithWorkspace | AppWithWorkspace;
 type DecoratedResource = DecoratedComputeResource | DiskWithWorkspace;
 
-interface EnvironmentsProps {
+export interface EnvironmentsProps {
   nav?: any;
 }
-export const Environments = ({ nav = undefined }: EnvironmentsProps) => {
+
+export const Environments: React.FC<EnvironmentsProps> = ({ nav = undefined }) => {
```

Comment:
A lot of code uses the existing form of type definitions for function components. I think adding React.FC would be something of a break with convention.

Reply:
https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components has a comparison of different styles of type declarations as well as some issues with `React.FC`.

Reply:
It doesn't matter so much now that we're on React 18, but based on the linked create-react-app issue and the incompatibility with generics that it mentions, I think I'd disagree with that recommendation.